### ISSUE TITLE
Artifactory HA broken on 7.84.10 release

### ISSUE THREAD

**Issue by jandersen-plaid** [jfrog/charts#1883]
**Is this a request for help?**: No.

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
Helm: 3.14.4
Kubernetes: 1.25.16

**Which chart**:
artifactory-ha

**Which product license** (Enterprise/Pro/oss):
Enterprise

**JFrog support reference** (if already raised with support team): 
N/A

**What happened**:
The helm templating is broken in release 7.84.10 because the template logic tries to reference `.Values` within a loop without using the global scope `$`:

https://github.com/jfrog/charts/blob/cef1e61d494d6f55f5bac8604947b24e554a5823/stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml#L1137-L1156

There is a bug there where `{{ .Values.artifactory.image.pullPolicy }}` should be `{{ $.Values.artifactory.image.pullPolicy }}` so that it uses the global scope.

Specifically, we receive the error:
```
Error: template: artifactory-ha/templates/artifactory-primary-statefulset.yaml:1140:35: executing "artifactory-ha/templates/artifactory-primary-statefulset.yaml" at <.Values.artifactory.image.pullPolicy>: can't evaluate field Values in type interface {}

Use --debug flag to render out invalid YAML
```

**What you expected to happen**:
We expected the helm chart to render successfully when loggers are set.

**How to reproduce it** (as minimally and precisely as possible):
To reproduce, you need to have `.Values.artifactory.loggers` set and run `helm template` with those values:

```yaml
artifactory:
  loggers: 
  - access-service.log
  - artifactory-binarystore.log
  - artifactory-event.log
  - artifactory-import-export.log
  - event-service.log
  - frontend-request.log
  - frontend-service.log
  - integration-service.log
  - metadata-request.log
  - metadata-service.log
  - router-request.log
  - router-service.log
```

**Anything else we need to know**: N/A

**Comment by chukka** [comment]
@jandersen-plaid Thanks for reporting ! we have internally fixed it and waiting for release

**Comment by chukka** [comment]
@jandersen-plaid Please try `107.84.11` chart for the fix and share feedback

**Comment by vgupta0309** [comment]
Was getting the same issue, but resolved after using chart version `107.84.11`
